src/ripplegw/waveforms/IMRPhenomD_NRTidalv2.py

```
@@ -622,3 +624,104 @@ def gen_IMRPhenomD_NRTidalv2_hphc(
+            *(
+                ("lambda_tilde", "delta_lambda_tilde")
+                if self.use_lambda_tildes
+                else ("lambda_1", "lambda_2")
+            ),
+            "d_L",
+            "phase_c",
+            "iota",
+        )
+
+    def __call__(
+        self, frequency: Float[Array, " n_freq"], params: Mapping[str, Any]
```

Ruff (0.15.21)

[error] Syntax error in forward annotation: Expected an expression

(F722)